### PR TITLE
Add some new posters

### PR DESCRIPTION
**Pull request by BeeRobynn**
## About the PR
Adds some new posters

## Media ##
image

**Changelog**
:cl:
- add: Added some new posters

### CODE REVIEW

**Comment by LevitatingTree** [comment]
no way you made this while i'm trying to figure out how to add my _own_ posters here as my first PR lmao, nice sprites

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml [hunk 1]
```diff
@@ -1068,6 +1068,89 @@
...
+    
+# Goob posters
+
+- type: entity
+  parent: PosterBase
+  id: PosterLegitBigBrotherAI
+  name: "AI Sees All"
+  description: "A poster that reminds you that your secrets are not hidden from your Station AI"
+  components:
+  - type: Sprite
```
[the remark this answers is not in the document]
Comment (Piras314) [reply Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml:1080]: We don't indent after components: in this codebase. Please remember this @coderabbitai

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml [hunk 2]
```diff
@@ -1068,6 +1068,89 @@
...
+  name: "Durk Plushie Advertisement"
+  description: "A poster depicting some sort of plush character being delivered soon to a station near you"
+  components:
+  - type: Sprite
+    state: poster54_legit
+
+- type: entity
+  parent: PosterBase
+  id: PosterLegitWelderBomb
+  name: "Danges of Fuel Tanks"
```
[the remark this answers is not in the document]
Comment (Piras314) [reply Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml:1095]: @BeeRobynn

**Comment by Lyacs** [comment]
That reminds me I should update the copyright on my own PR

File: Resources/Prototypes/Entities/Markers/Spawners/Random/posters.yml [hunk 3]
```diff
@@ -92,6 +92,8 @@
       - PosterContrabandInterdyne
       - PosterContrabandWaffleCorp
       - PosterContrabandMissingSpacepen
+      - PosterContrabandYakub
```
Comment (Piras314) [inline Resources/Prototypes/Entities/Markers/Spawners/Random/posters.yml:95]: ermmm... you need a `# Goobstation` comment

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml [hunk 4]
```diff
@@ -1068,6 +1068,89 @@
   components:
   - type: Sprite
     state: poster52_legit
+    
+# Goob posters
```
Comment (Piras314) [inline Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml:1072]: Erm.... these should go in `Resources/Prototypes/_Goobstation/Entities/Structures/Wallmounts/Signs/posters.yml 🤓 ☝️

File: Resources/Textures/Structures/Wallmounts/posters.rsi/poster53_legit.png
Comment (Piras314) [inline Resources/Textures/Structures/Wallmounts/posters.rsi/poster53_legit.png:1]: eeeeeeeeeeeeeeeeeeeeerrrrrrrrrrmmmmmmmmmmm.... these should go in `Resources/Textures/_Goobstation/Structures/Wallmounts/posters/posters.rsi

**Review by Piras314 — CHANGES_REQUESTED** [review]
🤓 👇

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml (hunk 4 above)
Comment (BeeRobynn) [reply Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml:1072]: Putting them in goob messes with file paths and I dont know the workaround for giving the markers two paths for sprites

Comment (Piras314) [reply Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml:1072]: you just specifically set the sprite within the sprite component to the goob rsi path

Comment (Piras314) [inline Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml:1072]: You forgot to delete these

File: Resources/Textures/Structures/Wallmounts/posters.rsi/meta.json [hunk 5]
```diff
@@ -1,439 +1,439 @@
...
-    {
-      "name": "lighthousemap"
-    },
-    {
-      "name": "waystationmap"
-    }
-  ]
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from  at commit https://github.com/tgstation/tgstation/commit/f01de25493e2bd2706ef9b0303cb0d7b5e3e471b.     poster52_contraband, poster53_contraband and poster54_contraband taken from https://github.com/vgstation-coders/vgstation13/blob/435ed5f2a7926e91cc31abac3a0d47d7e9ad7ed4/icons/obj/posters.dmi.     originmap, poster55_contraband, poster56_contraband, poster57_contraband and poster39_legit by discord brainfood#7460,     poster63_contraband by discord foboscheshir_,     poster52_legit by SlamBamActionman,     53_legit - 59_legit, 64_contraband lovingly handmade by BeeRobyn on Discord - https://github.com/BeeRobynn, 59_legit created by wyvernrer on Discord, for 65_contraband Yakub came to me in my dreams and gave me the sprite",
```
Comment (Piras314) [inline Resources/Textures/Structures/Wallmounts/posters.rsi/meta.json:4]: revert your changes to this file

**Review by Piras314 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**